package.json

```diff
@@ -53,11 +54,11 @@
     "dependencies": {
         "he": "^1.2.0",
         "html-parser": "^0.11.0",
-        "ilib": "^14.17.0",
+        "ilib": "^14.18.0",
         "js-stl": ">=0.0.6"
     },
     "devDependencies": {
-        "loctool": "^2.20.2",
-        "nodeunit": "^0.11.3"
+        "jest": "^26.6.3",
```

Comment:
oh,  When I checked the latest version of jest, It's `29.7.0`. Is there any specific reason you picked the major version 26?

Reply:
The latest version of `jest` doesn't work properly on node v12

Reply:
Yes, exactly, or on node 10. But, loctool still supports node 10 and 12